plugins/discovery-gce/build.gradle

```diff
@@ -17,22 +17,23 @@ opensearchplugin {
   classname 'org.opensearch.plugin.discovery.gce.GceDiscoveryPlugin'
 }
 
-versions << [
-  'google': '1.23.0'
-]
-
 dependencies {
-  api "com.google.apis:google-api-services-compute:v1-rev160-${versions.google}"
-  api "com.google.api-client:google-api-client:${versions.google}"
+  api "com.google.apis:google-api-services-compute:v1-rev235-1.25.0"
```

Comment:
@cwperks is `${versions.google}` used for anything now?

Reply:
@reta See the lines above that were removed. `versions.google` is not coming from `buildSrc/version.properties`

Reply:
Ah indeed, thanks you!